ShelterModule/Models/Pets/Pet.cs

```diff
@@ -1,78 +1,75 @@
 ﻿using Database.Entities;
-using ShelterModule.Models.Shelters;
 
-namespace ShelterModule.Models.Pets
+namespace ShelterModule.Models.Pets;
+
+public class Pet
 {
-    public class Pet
-    {
-        public Guid Id { get; init; }
-        public required Shelter Shelter { get; init; } = null!;
-        public required string Name { get; init; } = null!;
-        public required string Species { get; init; } = null!;
-        public required string Breed { get; init; } = null!;
-        public required DateTime Birthday { get; init; }
-        public required string Description { get; init; } = null!;
-        public required string Photo { get; init; } = null!;
+    public Guid Id { get; init; }
+    public required Guid ShelterId { get; init; }
```

Comment:
I've removed `Shelter` from this class and left just `ShelterId`, as we don't use the whole shelter anyway, only the ID.